Makefile

```diff
@@ -29,6 +41,53 @@ run-test:
 	@cd $(root_path)/code/go/0chain.net && go test -tags bn256 -cover ./...
 	@echo "Tests completed."
 
+install-mockery:
+	@[ -d ./bin/mockery ] || mkdir -p ./bin/mockery
```

Comment:
Please do not install the executable file to the project, install it to $GOPATH/bin or $HOME/go/bin, the same as go install/get

Reply:
@peterlimg  it is temp install dir, and is ignored by .gitignore. the mockery will be installed to $GOPATH/bin by `	@cp ./bin/mockery/mockery $(GOPATH)/bin/`.   the temp dir will save time to next `make build-mocks`.